src/main/scala/org/broadinstitute/dsde/workbench/avram/api/BaseEndpoint.scala

```diff
@@ -1,11 +1,21 @@
 package org.broadinstitute.dsde.workbench.avram.api
 
+import java.util.logging.Logger
 import javax.servlet.http.HttpServletRequest
+
 import org.broadinstitute.dsde.workbench.avram.Avram
 import org.broadinstitute.dsde.workbench.avram.dataaccess.SamUserInfoResponse
+import org.broadinstitute.dsde.workbench.avram.db.DataAccess
 import org.broadinstitute.dsde.workbench.avram.util.ErrorResponse
+import slick.dbio.DBIO
+
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
 
 abstract class BaseEndpoint {
+
+  private val log = Logger.getLogger(getClass.getName)
+  val database = Avram.database
```

Comment:
The way that `AvramRoutes.dbPoolStats` uses this `database` property is definitely non-standard. Endpoints should be using the provided `inTransaction` function. Therefore, this `database` property should be private and `AvramRoutes.dbPoolStats` should be forced to access `Avram.database` directly.

This probably seems a bit picky but I think it's worth doing to establish the correct patterns now.